config/initializers/rails_admin.rb

```diff
@@ -146,29 +142,10 @@
           "#{self.name}_field sponsorship_fields_toggler"
         end
       end
-      configure :challenge_categories do
-        hide
-      end
-      configure :submitted_flags do
-        hide
-      end
-      configure :solved_challenges do
-        hide
-      end
-      configure :unsolved_increment_period do
-        hide
-      end
-      configure :unsolved_increment_points do
-        hide
-      end
-      configure :initial_shares do
-        hide
-      end
-      configure :solved_decrement_shares do
-        hide
-      end
-      configure :first_capture_point_bonus do
-        hide
+      [:challenge_categories, :submitted_flags, :solved_challenges, :unsolved_increment_period, :unsolved_increment_points, :initial_shares, :solved_decrement_shares, :first_capture_point_bonus].each do |field|
```

Comment:
I know we have rubocop disabled on initializers but I would like to still keep the line length less than this if possible. It should be possible to write this as 
```ruby
[
  :challenge_categories,
  :submitted_flags,
  ...
].each do |field|
  ...
end

`